microsoftgraph/microsoft-graph-explorer-v4
# Fix: Refactor sample queries

## Overview

Fixes #1676 

## Testing Instructions

* Load GE
* Navigate to sample queries 
* Try all kinds of queries
* Try the search box
* Click on icons 
etc.. ensure it works as it previously did
